packages/backend/src/tokens/interstitialRule.ts

```diff
@@ -126,11 +126,11 @@ export async function runInterstitialRules<T>(opts: T, rules: InterstitialRule[]
   return signedOut<T>(opts, AuthErrorReason.UnexpectedError);
 }
 
-async function verifyRequestState(options: any, token: string) {
-  const issuer = (iss: string) => iss.startsWith('https://clerk.') || iss.includes('.clerk.accounts');
+async function verifyRequestState(options: any, token: string, issuer = '') {
```

Comment:
⛔ My recommendation is to avoid changing the signature of `verifyRequestState` method and pass the issuer as another key in the options. The options argument was introduced to cater for such cases.

Reply:
@SokratisVidros so you recommend removing the `issuer` param from `verifyRequestState` and using the `option.proxyurl` straight away right?

Reply:
Yes